docs/README.md

```diff
@@ -9,5 +9,21 @@ It is possible to wrap a React element with [Overlay](https://github.com/vtex-ap
 ```
 <Overlay>
   <div>Hello World!</div>
-</Overlay> 
+</Overlay>
 ```
+
+## Props
+
+| Prop name    | Type                  | Description                                                  | Default value |
+| ------------ | --------------------- | ------------------------------------------------------------ | ------------- |
+| `alignment`  | `HorizontalAlignment` | Horizontal alignment                                         | `left`        |
+| `fullWindow` | `Boolean`             | If true, the overlay will fill the whole window horizontally | -             |
```

Comment:
Maybe use `fullScreen` instead?

Reply:
I see no problem with that!

But if I change this prop name it will create a break change since `fullWindow` was already declared in this project before this PR. Is it ok?

Reply:
Oh, no. so leave it be